src/crystal/project/dsl.cr

```diff
@@ -13,5 +13,9 @@ struct Crystal::Project::DSL
     def github(repository, name = nil : String)
       @project.dependencies << GitHubDependency.new(repository, name)
     end
+
+    def local(path, name = nil : String)
```

Comment:
Ah, I just see that. Yes, I think "path" would be a better name. @naps62 what do you think?